src/main/ruby/core/rbconfig.rb

```diff
@@ -80,6 +83,7 @@ module RbConfig
       'NULLCMD'           => ':',
       'optflags'          => '',
       'RM'                => 'rm -f',
+      'PATH_SEPARATOR'    => ':',
```

Comment:
This is a platform-dependent value, isn't it? If so, it'd be good to reflect that here.

Reply:
It could use `File::PATH_SEPARATOR`